DependencyInjection/SonataDoctrinePHPCRAdminExtension.php

```diff
@@ -89,6 +89,18 @@ public function getNamespace()
      */
     private function loadTreeTypes($config, ContainerBuilder $container)
     {
+        @trigger_error(
+            'The "sonata_doctrine_phpcr_admin.document_tree_options" configuration is deprecated since 1.2.8, to be replaced in 2.0.',
+            E_USER_DEPRECATED
+        );
+
+        if (isset($config['document_tree_defaults'])) {
+            @trigger_error(
+                'The "sonata_doctrine_phpcr_admin.document_tree_defaults" configuration is deprecated since 1.2.8, to be moved in 2.0.',
+                E_USER_DEPRECATED
+            );
+        }
```

Comment:
This PR is supposed to introduce the thing you are deprecating this in favor of if it is not already available. The PR on master should only do removals.